bedrock/firefox/templates/firefox/features/adblocker.html

```diff
@@ -0,0 +1,159 @@
+
+ {% extends "base-protocol.html" %}
+
+ {% block page_title %}{{ _('Firefox Adblocking Feature') }}{% endblock %}
+
+ {% block page_desc %}
+   {{ _('Mozilla makes browsers, apps, code and tools that put people before profit. Our mission: Keep the internet open and accessible to all.') }}
+ {% endblock %}
+
+ {% block page_css %}
+   {{ super() }}
+   {{ css_bundle('adblocker') }}
+ {% endblock %}
+
+ {% block content %}
+
+  <section class="mzp-c-hero mzp-t-product-firefox">
+    <div class="mzp-l-content">
+      <div class="mzp-c-hero-body">
+        <h1 class="mzp-c-hero-title">{{ _('Everything you need to know about ad blockers.') }}</h1>
+        <div class="mzp-c-hero-desc">
+          <p>{{ _('Ad blocker is a web browser extension-- that can detect web ads and prevent them from being displayed on a web page.') }}</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="mzp-l-content l-content-image">
+    {{ high_res_img('firefox/features/privacy-illustration1.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
+  </section>
+
+  <section class="mzp-l-content l-content-narrow">
+    <h2 class="section-title">{{ _('How ad blockers work') }}</h2>
+    <div class="l-content-columns">
+      <div>
+        <h3>{{ _('A typical ad blocker') }}</h3>
+        <p>{{ _('Most ad blockers work in two ways.') }}</p>
+        <p>{{ _('First, they block communication from ad servers or others. Ads simply can’t get through.') }}</p>
+        <p>{{ _('Second, they hide parts of a web page that could be ads-- any section of a site that’s labeled "ad," for example. Most ad blockers use public and up-to-date lists of known ad servers.') }}</p>
+      </div>
+      <div>
+        <h3>{{ _('The Firefox ad blocker') }}</h3>
+        <p>
+          {% trans url='https://blog.mozilla.org/firefox/tracking-protection-always-on/' %}
+          Firefox doesn’t include ad blocking out of the box, but it does include <a href="{{ url }}">Tracking Protection</a>, which stops sites from installing those tracking cookies we talked about.
+          {% endtrans %}
+        </p>
+        <p>
+          {% trans url='https://support.mozilla.org/en-US/kb/tracking-protection' %}
+          It’s <a href="{{ url }}">easy to turn on</a>, and has the added benefit of making Firefox significantly faster.
+          {% endtrans %}
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="mzp-l-content l-content-narrow">
+    <h2 class="section-title">{{ _('Why use ad blockers?') }}</h2>
+    <p>{{ _('Ad blockers can also be used to make the web faster.') }}</p>
+    <p>{{ _('Ads can slow down sites while they load, or get in the way of a streamlined browsing experience.') }}</p>
+    <p>{{ _('Ads can also be flashy and annoying, distracting you from the stuff you really want to see.') }}</p>
+    <blockquote>{{ _('“Ads can slow down sites… or get in the way of a streamlined browsing experience.”') }}</blockquote>
```

Comment:
I don't see any styling on the blockquote. Is that coming as part of a protocol import I'm missing or something?

Reply:
Most likely! Craig/Alex merged #6232 which includes new blockquote styles. You may need to rebase? 🤷‍♀️

Reply:
I did a `make build` and they showed up 👍